composer/build.gradle

```diff
@@ -21,7 +24,7 @@ dependencies {
 }
 
 jar {
-    // Build "fat" jar.
+    // Build jar with dependencies.
```

Comment:
Well, I was afraid that it might offend someone. Same as master-slave -> master-nodes. `with-dependencies` is used in few other projects I used.